introduction.md

```diff
@@ -1,2 +1,6 @@
 Introducing dnshl
-My favorite Code School path is Electives
\ No newline at end of file
+My favorite Code School path is Electives
+
+* Git
+* JavaScript
+* Ruby
\ No newline at end of file
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?